pyproject.toml

```diff
@@ -50,6 +50,7 @@ dev = [
     "ipython", # Also used in building notebooks into Sphinx
     "matplotlib", # Used in sample notebook intro_notebook.ipynb
     "ray", # Used for dask-on-ray testing.
+    "types-PyYAML", # type stubs for pyyaml
```

Comment:
does this also include the import for `pyyaml`?

Reply:
Hm. It's transitively included from the `dask` dependency. I'll add as explicit dependency.